### DISCUSSION TITLE
400 Bad Request when loading client library

### DISCUSSION THREAD

**Discussion by 56kj** [abacritt/angularx-social-login#705] [Q&A]
Trying to implement google sign in, and I noticed I am seeing the same error that occurs when viewing the console of the demo page: https://abacritt.github.io/angularx-social-login/. Any idea what impact this has and if this is expected behavior? Or how to fix?

GET | https://accounts.google.com/gsi/status?client_id=624796833023-clhjgupm0pu6vgga7k5i5bsfp6qp6egh.apps.googleusercontent.com&as=GlRTIYUlCqwbJqQvZFeeQw

400
Bad Request

Response:
)]}'
[["GlRTIYUlCqwbJqQvZFeeQw"],[2]]

**Comment by YashrajDawkhar** [comment]
same issue
